types/slonik/slonik-tests.ts

```diff
@@ -24,6 +24,11 @@ import {
   TypeParserType,
   UniqueIntegrityConstraintViolationError
 } from 'slonik';
+import { ArrayTokenSymbol, TupleListTokenSymbol } from 'slonik/symbols';
+
+// make sure symbols are unique
+// $ExpectError
+const badSymbolAssignment: typeof ArrayTokenSymbol = TupleListTokenSymbol;
```

Comment:
previously all the symbols were assignable to each other - I updated them from `symbol` to `unique symbol` and this is the test to make sure that sticks